contracts/common/MetaTransactionMsgSender.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.7.3;
+
+import "../../lib/dappsys/math.sol";
+
+abstract contract MetaTransactionMsgSender is DSMath {
```

Comment:
Why not include this function in `BasicMetaTransaction`?

Reply:
Any contract that inherits `BasicMetaTransaction` needs to define the virtual functions, and `msgSender` is needed basically everywhere, so split up the functionality between `BasicMetaTransaction` (functionality required to execute a metatransaction) and `MetaTransactionMsgSender` (the functionality required to support metatransactions) in order to reduce the repetition of the virtual function declarations.